backend/support/models.py

```diff
@@ -28,7 +29,7 @@ class CognizantBaseline(models.Model):
     date_assigned = models.DateTimeField(
         # help_text = "Time when the cog agency was assigned to the entity",
         null=True,  # allow nulls in case history has nulls
-        auto_now_add=True,
+        default=timezone.now,
```

Comment:
Since this is addressing a bug that was discovered, we should include a test case to make sure the fix is resolving the bug.

Reply:
run_2022 command confirms that this fix is working.

Reply:
test_cog_over already has test cases.  The CognizantBaseline table is updated in signals.py.  The signal event is asynchronous and hence the date cannot be checked in test_cog_over.